opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -291,7 +291,7 @@ impl<RT: Runtime> PeriodicReaderWorker<RT> {
         true
     }
 
-    async fn run(mut self, mut messages: impl Unpin + FusedStream<Item = Message>) {
+    async fn run(mut self, mut messages: impl FusedStream<Item = Message> + Unpin) {
```

Comment:
curious - Is this change related to hyper upgrade?

Reply:
They're equivalent, but I do prefer the new version -- better to keep "boring" traits at the end.

Reply:
Yeah, that was part of the intention here. Also I had some linter issues with clippy on nightly lints a while back, I think?
Can't remember, but this is probably an artifact of that.